airbyte-webapp/src/pages/SettingsPage/pages/DbMigrationPage/components/MigrationTableView.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { DbMigrationInfoItem } from "core/resources/DbMigration";
+import { Block } from "../../ConnectorsPage/components/PageComponents";
```

Comment:
Please try to avoid relative imports if it is not within `./` directory. It makes it very difficult to refactor later